camelot/backends/pdfium_backend.py

```diff
@@ -0,0 +1,18 @@
+try:
+    import pypdfium2 as pdfium
+except Exception as e:
+    pdfium = None
+    pdfium_exc = e
+else:
+    pdfium_exc = None
+
+
+class PdfiumBackend:
+    def convert(self, pdf_path, png_path, resolution=300):
+        if not pdfium:
+            raise OSError(f"pypdfium2 is not available: {pdfium_exc!r}")
+        doc = pdfium.PdfDocument(pdf_path)
+        assert len(doc) == 1
```

Comment:
FWIW, I don't think it's a good design to split multi-page documents into 1-page parts as camelot seems to do, instead you had better track the page index and pass that to the PDF engine in question.